replace-http-with-https.sh fails to replace website URLs when using a Service Account with access to multiple vaults

### Problem
[Service accounts require a `--vault` flag if it has access to more than one vault.](https://developer.1password.com/docs/service-accounts/get-started/#supported-commands) As a result, unexpected issues occur when attempting to use `replace-http-with-https.sh`.

### Expected behavior
The "http" in the "website" field within a 1Password login item is replaced with "https".

### Actual Behavior

`[ERROR] <date> <time> a vault query must be provided when this command is called by a service account. Please specify one either through the --vault flag or through piped input`

"website" value for all relevant items are erased:
![CleanShot 2023-08-30 at 10 21 02](https://github.com/1Password/solutions/assets/14211202/9d504dc5-2a2d-41db-a939-e91918408dfd)

### Environment
CLI 2.20
Ubuntu 20.04.2 ARM64


### Suggestion
Add a '--vault' flag to each `op item` call in the script:
```
#!/usr/bin/env bash
#########################################################
# REPLACE HTTP WITH HTTPS 
#########################################################
# This script will replace "http" with "https" for the 
# website field of all Login items in the specified vault 

#Provide a vault UUID or vault name
vaultUUID=""

# add new url to each item
for item in $(op item list --vault $vaultUUID --categories Login --format json | jq --raw-output '.[].id')
do
	oldURL=$(op item get $item --vault $vaultUUID --format=json | jq --raw-output '.urls[].href')
	newURL=$(echo $oldURL | sed 's/^http:\/\//https:\/\//g')
	op item edit $item --vault $vaultUUID website=$newURL
done
```

--- a/item-management/replace-http-with-https.sh
+++ b/item-management/replace-http-with-https.sh
@@ -11,7 +11,7 @@ vaultUUID=""
 # add new url to each item
 for item in $(op item list --vault $vaultUUID --categories Login --format json | jq --raw-output '.[].id')
 do
-	oldURL=$(op item get $item --format=json | jq --raw-output '.urls[].href')
+	oldURL=$(op item get $item --vault $vaultUUID --format=json | jq --raw-output '.urls[].href')
 	newURL=$(echo $oldURL | sed 's/^http:\/\//https:\/\//g')
-	op item edit $item website=$newURL
+	op item edit $item --vault $vaultUUID website=$newURL
 done

